Assets/Scripts/Throw/NextColor.cs

```diff
@@ -47,9 +56,9 @@ public void Remove()
 
     private void Add()
     {
-        ColorSO nextColor = GetRandomColorSO();
+        ColorSO nextColor = ThreeColors();
         GameObject go = nextColor.YarnPrefab;
-        NextColorQueue.Enqueue(nextColor);
```

Comment:
Don't remove this, as it gives an error in the console when running without randomness, since that is how we add yarn balls to be thrown. Snippet of error given:
```
InvalidOperationException: Queue empty.
System.Collections.Generic.Queue`1[T].Peek ()
```